Wordle.Api/Wordle.Api/Models/Player.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Wordle.Api.Models;
+
+public class Player
+{
+    public int PlayerId { get; set; }
+    [Required]
+    public required string Name { get; set; }
+    public int GameCount { get; set; }
+    public double AverageAttempts { get; set; }
+    public double AverageSeconds { get; set; }

```

Comment:
I believe average seconds was supposed to be int

Reply:
Thanks for the note. Double makes more sense to me for an average, but I could be wrong.